pages.zh/common/docker.md

```diff
@@ -0,0 +1,36 @@
+# docker
+
+> 管理 Docker 容器和镜像。
+> 更多信息： <https://docs.docker.com/engine/reference/commandline/cli/>.
+
+- 列出目前正在运行的 docker 容器:
+
+`docker ps`
+
+- 列出所有 docker 容器（包括停止的容器）：
+
+`docker ps -a`
+
+- 透过镜像启动容器，并为容器命名：
+
+`docker run --name {{容器名称}} {{镜像}}`
+
+- 启动或停止现有容器：
+
+`docker {{start|stop}} {{容器名称}}`
```

Comment:
I think we can list more command here, such as `restart`

Reply:
Sounds good, but that should be done in a separate PR and applied to all translations.